docs/datasets/README.md

```diff
@@ -0,0 +1,43 @@
+# Generating an AIP database
+
+Disclaimer: to update the database one would need around 1TB of storage available and about 30h of computation time. 
+
+In order to update the database you must download all 4 data sources and parse them. 
```

Comment:
3, right? DBLP, Aminer, Semantic Scholar. Unless MAG indeed was added (I think it was), then 4 is correct.

Reply:
Yes, it should be 4